ts-obs/tests/main_test.go

```diff
@@ -0,0 +1,89 @@
+package tests
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"testing"
+)
+
+func startKube() {
+	var err error
+
+	log.Println("Starting Kubernetes cluster")
+
+	kubestart := exec.Command("minikube", "start")
+	err = kubestart.Run()
+	if err != nil {
+		log.Println("Error starting Kubernetes cluster", err)
+		os.Exit(1)
+	}
+}
+
+func installObs() {
+	var err error
+
+	log.Println("Installing Timescale Observability")
+
+	obsinstall := exec.Command("ts-obs", "install")
```

Comment:
I think the tests will more likely find bugs if our main install uses a non-default namespace and non-default release name